app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -6,7 +6,7 @@ class ParticipantDeclarationsController < Api::ApiController
       include ApiTokenAuthenticatable
 
       def create
-        params = HashWithIndifferentAccess.new({ raw_event: request.raw_post, lead_provider_from_token: cpd_lead_provider }).merge(permitted_params["attributes"] || {})
+        params = HashWithIndifferentAccess.new({ lead_provider_from_token: cpd_lead_provider }).merge(permitted_params["attributes"] || {})
```

Comment:
We need this to protect against schema API changes for declarations. It keeps us immutable and protects against API changes.